commerce-payment-test/src/testIntegration/java/com/liferay/commerce/payment/engine/test/CommercePaymentEngineTest.java

```diff
@@ -216,6 +246,18 @@ public void testProcessPayment() throws Exception {
 	@Rule
 	public FrutillaRule frutillaRule = new FrutillaRule();
 
+	@DeleteAfterTestRun
+	private CommerceChannel _commerceChannel;
+
+	@Inject
+	private CommerceChannelLocalService _commerceChannelLocalService;
+
+	@DeleteAfterTestRun
+	private CommerceChannelRel _commerceChannelRel;
+
+	@Inject
+	private CommerceChannelRelLocalService _commerceChannelRelServ;
```

Comment:
Should be named `CommerceChannelRelLocalService _commerceChannelRelLocalService`, for clarity/consistency.